### PR TITLE
[ .kokoro] Added todo for error encountered

### PR DESCRIPTION
**Pull request by randallli**
I encountered an error with a new computer that I want redirect to the solution.

This PR still needs to be massaged to make running kokoro emit the suggested fix.

### CODE REVIEW

File: .kokoro [hunk 1]
```diff
@@ -271,6 +271,12 @@ run_bazel() {
   selected_xcode_developer_path=$(xcode-select -p)
   selected_xcode_contents_path=$(dirname "$selected_xcode_developer_path")
 
+  # todo fix error about missing plist
+  # 
+  # Running bazel builds...
+  # cat: /Library/Developer/version.plist: No such file or directory
+  #
+  # fixed by choosing command line tool from xcode.
```
Comment (yarneo) [inline .kokoro:279]: what do you mean by choosing command line tool from xcode?

Comment (randallli) [reply .kokoro:279]: https://www.embarcadero.com/starthere/xe5/mobdevsetup/ios/en/installing_the_commandline_tools.html

File: .kokoro [hunk 2]
```diff
@@ -271,6 +271,12 @@ run_bazel() {
   selected_xcode_developer_path=$(xcode-select -p)
   selected_xcode_contents_path=$(dirname "$selected_xcode_developer_path")
 
+  # todo fix error about missing plist
```
Comment (romoore) [inline .kokoro:274]: Tip: Please use `TODO(b/1234):` format to make this easier to track internally.

File: .kokoro (hunk 1 above)
Comment (romoore) [reply .kokoro:279]: Is this different from `xcode-select`?

Comment (randallli) [reply .kokoro:279]: probably not different from xcode-select

**Comment by jverkoey** [comment]
Closing this PR for now, but please re-open it if/when you are working on it.

**Pull request closed without merging**